.github/workflows/modelhub_tests.yml

```
@@ -63,6 +63,10 @@ jobs:
         run: |
           cd modelhub
           pycodestyle modelhub
+      - name: Unit tests
+        run: |
+          cd modelhub
+          pytest tests_modelhub/unit/  -W error::DeprecationWarning
```

probably is safe to just leave it like this, anyways we should remove it from ``bach_tests.yml`` too
```suggestion
          pytest tests_modelhub/unit/ 
```